test/unit/job_invocation_test.rb

```diff
@@ -0,0 +1,44 @@
+require 'test_plugin_helper'
+
+describe JobInvocation do
+
+  let(:job_invocation) { FactoryGirl.build(:job_invocation) }
+  let(:template) { FactoryGirl.build(:job_template) }
+
+  context 'Able to be created' do
+    it { assert job_invocation.save }
```

Comment:
what do we test here? if it's just validation, you can use assert_valid from Foreman core test_helper.rb which doesn't trigger DB query